src/main/java/sirius/biz/storage/layer3/VirtualFileSystem.java

```diff
@@ -200,11 +200,6 @@ public String makePath(String... parts) {
      */
     public boolean isAutoCleanupBlob(VirtualFile virtualFile) {
         Optional<BlobStorageSpace> blobStorageSpace = virtualFile.tryAs(BlobStorageSpace.class);
-
-        if (blobStorageSpace.isEmpty()) {
-            return false;
-        }
-
-        return blobStorageSpace.get().getRetentionDays() > 0;
+        return blobStorageSpace.filter(storageSpace -> storageSpace.getRetentionDays() > 0).isPresent();
```

Comment:
Could even collapse this to one line, by removing the variable declaration and directly returning `virtualFile.tryAs(...).filter(...).isPresent()`